evervault/http/request.py

```diff
@@ -33,6 +33,10 @@ def __build_headers(self, method, params, version):
             "Content-Type": "application/json",
             "Api-Key": self.api_key,
         }
+        if "x-async" in optional_headers:
+            headers["x-async"] = optional_headers["x-async"]
+        if "x-version-id" in optional_headers:
+            headers["x-version-id"] = optional_headers["x-version-id"]
```

Comment:
Suggestion, let's just merge the headers together instead of checking, so we can add our own headers when running the SDK for testing purposes locally?